chia/full_node/full_node.py

```diff
@@ -1122,9 +1124,7 @@ async def validate_block_batches(
                     # accrue a large state (with no value, since we can validate
                     # against the CoinStore)
                     if not extending_main_chain:
-                        if fork_point_height == 0:
-                            fork_info = ForkInfo(-1, -1, bytes32([0] * 32))
```

Comment:
It's not obvious to me why you removed this. I would expect this to cause problems

Reply:
from what i understand its not used, we use None for cases where we dont know the fork_point no?

Reply:
i think i found the edge case where this is used, its used to diff when we sync from 0 to when we dont have a fork point, it seems a little patchy that we have both the None case and the initialized to -1 case but ill leave it